### PR TITLE
Update mesos docs

### PR DESCRIPTION
**Pull request by jzelinskie**

### CODE REVIEW

File: issues/quay-mesos.md [hunk 1]
```diff
@@ -4,9 +4,6 @@ sublayout: issue
 title: Pulling Private Quay.io images with Marathon/Mesos fails
 frontpage: True
 ---
-
-#### Special thanks to [Ian Saunders](https://github.com/IanSaunders) who provided this solution.
```
Comment (josephschorr) [inline issues/quay-mesos.md:8]: Don't remove this please.

File: issues/quay-mesos.md [hunk 2]
```diff
@@ -15,12 +12,17 @@ msg="Error: Status 403 trying to pull repository repo/project: \"{\\\"error\\\":
...
 
-To get Marathon/Mesos to pull from a private repository, we need to copy the `.docker/config.json`
-credentials onto the worker machines.
+To get Marathon/Mesos to pull from a private repository, we need to copy the docker configuration file's credentials onto the worker machines.
+
+When using mesos app definitions, credentials must be provided as a URI that must be accessible by all nodes that may start your application.
+Approaches may include distributing the file to the local filesystem of all nodes, for example via RSYNC/SCP, or storing it on a shared network drive, for example Amazon S3.
+It is worth considering the security implications of each approach.
+
+Special thanks to [Ian Saunders](https://github.com/IanSaunders) who provided this solution.
```
Comment (josephschorr) [inline issues/quay-mesos.md:21]: I still prefer this at the top

File: issues/quay-mesos.md (hunk 1 above)
Comment (jzelinskie) [inline issues/quay-mesos.md:8]: The first line is always a description of the problem and if we end up getting other solutions not from him, it doesn't make sense to have his name at the top of the article, but in the section that he actually contributed. You said this was okay when I asked yesterday.

Comment (josephschorr) [inline issues/quay-mesos.md:8]: I don't recall that, but the reasoning is sound, so SGTM

**Comment by josephschorr** [comment]
LGTM

**Pull request merged**